src/Converter.php

```diff
@@ -29,6 +29,25 @@ public function getAsText(): string
         return $output;
     }
 
+    public function getSubstringHtml(int $numberOfCharacters, bool $escape = false): string
+    {
+        $output = '';
+        $checkOutput = '';
+        
+        foreach ($this->rows as $row) {
+            $checkOutput .= $row;
+
+            if ( strlen($checkOutput) > $numberOfCharacters ){
```

Comment:
These random spaces make me guess that you forgot to run php-cs-fixer...

Reply:
Yep i've ran cs-fixer